docs/field-type-validator-usage.md

```diff
@@ -0,0 +1,73 @@
+# Field Type Validator
+
+Field Type Validator is a base class to validate inputs in field type.
+
+## How to use it?
+
+```javascript
+class Validator extends global.eZ.BaseFieldValidator {
+    validateInput(event) {
+        // validation
+
+        return {
+            isError: {Boolean},
+            errorMessage: {String},
+        };
+    }
+};
+
+const validator = new Validator({
+    classInvalid: {String},
+    fieldSelector: {String},
+    eventsMap: [
+        {
+            selector: {String},
+            eventName: {String},
+            callback: {String},
+            invalidStateSelectors: {Array},
+            errorNodeSelectors: {Array},
+        },
+    ],
+});
+
+validator.init();
+```
+
+## Props list
+
+The `BaseFieldValidator` class has a few required properties. All of them are listed below.
+
+**classInvalid** _{String}_ - CSS class to be added when field is invalid, example: 'is-invalid'.
+
+**fieldSelector** _{String}_ - selector of the field, example: '.ez-field-edit-ezstring',
+
+**eventsMap** _{Array}_  - events config:
+- **selector** _{String}_ - selector of input (where add event listener), example: '.ez-field-edit-ezstring input'.
```

Comment:
`- **selector** _{String}_ - selector of input (where to add an event listener), example: '.ez-field-edit-ezstring input'.`